src/marten/conf/global_settings.cr

```diff
@@ -256,12 +256,17 @@ module Marten
         @csrf ||= GlobalSettings::CSRF.new
       end
 
-      # Allows to configure a specific database connection for the application.
-      def database(id = DB::Connection::DEFAULT_CONNECTION_NAME, &)
+      def database(id = DB::Connection::DEFAULT_CONNECTION_NAME, url : String | Nil = nil)
```

Comment:
@ellmetha might be missing something with the overloading as it's my first time fiddling with crystal, but I think we have to stick with `url` as a named parameter here, therefore configuring with `config.database url: "postgres://my_user:my_db@localhost:1234/db"`